blog_system/settings.py

```diff
@@ -0,0 +1,137 @@
+"""
+Django settings for blog_system project.
+
+Generated by 'django-admin startproject' using Django 4.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+from django.conf import settings
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-%mp)6e6)*7j$zyj8k&p6m$5ex^-2-9%zpf1dlioyf_duw7xmb("
```

Comment:
The `SECRET_KEY` is hardcoded in the settings file. For security reasons, especially in production, it's recommended to use environment variables to manage sensitive information like the secret key.